procstats/proc.go

```diff
@@ -133,16 +133,25 @@ func (p *ProcMetrics) Collect() {
 		now := time.Now()
 
 		if !p.lastTime.IsZero() {
-			interval := now.Sub(p.lastTime)
+			ratio := 1.0
```

Comment:
I'm probably mis-understanding something here. But it seems that `ratio` will be the number of cores if we are using a quota or shares. Should we be using the number of CPUs for the default use case ?

Reply:
That's a good idea. I'm not sure what's more _natural_ to reason about, for example having 100% be 1 core, and when it's using more it goes over 100 (kind of like top does it), or normalizing based on the number of cores available.